docs/reference/run.md

```diff
@@ -823,6 +824,11 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The `--blkio-weight-device` flag can set the weighting of a particular device to a value between 10 to 1000.
```

Comment:
It would be helpful to explain how `blkio-weight` and `blkio-weight-device` interact.